bin/gssstdio.c

```diff
@@ -472,7 +472,7 @@ read_packet(int fd, gss_buffer_t buf, int timeout, int first)
 		    timeout);
 
 		if (ret == -1) {
-			if (errno == EINTR || errno == EAGAIN)
+			if (errno==EINTR||errno==EAGAIN||errno==EWOULDBLOCK)
```

Comment:
We might need a

```
#ifndef EWOULDBLOCK
#define EWOULDBLOCK EAGAIN
#endif
```

Reply:
As discussed, will add this later if we determine this is needed.